docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_slider.htm

```diff
@@ -1,58 +1,480 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
 <html>
- <head>
-     <script type="text/javascript" src="../snippet.js"></script>
-     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-     <link href="../styles.css" rel="StyleSheet" type="text/css" />
-     <link href="../snippet.css" rel="StyleSheet" type="text/css" />
-     <title>Slider</title>
- </head>
- <body id="content">
+<head>
+	<script type="text/javascript" src="../snippet.js"></script>
+	<meta http-equiv="content-type" content="text/html; charset=utf-8">
```

Comment:
Question. Was there any reason for changing "/>" to ">"?

Reply:
@pkaczmarczy

Reply:
This documentation page has been built based on mobile slider documentation and header has been copied.
However <meta> and <link> are void elements, according to https://www.w3.org/TR/html5/syntax.html#start-tags item 6 the "/"  is optional and it has no effect.

Reply:
Thanks for the description.
@TomaszLukawskiSam , please check the latest version and approve it if this is no issue in this PR.

Reply:
I accept @pkaczmarczy 's explanation. I accept PR.